src/net/sourceforge/kolmafia/maximizer/Evaluator.java

```diff
@@ -2657,10 +2657,10 @@ else if ( total < useful )
 				}
 			}
 
-			// Blunt object fix for only having a foldable that might be neede elsewhere
+			// Blunt object fix for only having a foldable that might be needed elsewhere
 			if ( automatic[ slot ].size() == 1 && ItemDatabase.getFoldGroup( automatic[ slot ].get( 0 ).getName() ) != null )
 			{
-				automatic[ slot ].add( new CheckedItem( 0, equipScope, maxPrice, priceLevel ) );
+				automatic[ slot ].add( new CheckedItem( -1, equipScope, maxPrice, priceLevel ) );
```

Comment:
I only found two instances of calling CheckedItem with an integer constant instead of a variable and it looks like you got them both.

The part of me that doesn't like Magic Numbers  will go and suppress itself :-)

Reply:
I don't like magic numbers either! Sentinel values are evil, and all that.